draft-ietf-tls-tls13.md

```diff
@@ -1106,7 +1106,7 @@ and in TLS 1.3 the additional data input is empty (zero length).
 opaque_type
 : The outer opaque_type field of a TLSCiphertext record is always set to the
   value 23 (application_data) for outward compatibility with
-  middleboxes used to parsing previous versions of TLS.  The
```

Comment:
The current line is correct English. If we would like to simplify the language here, the suggested replacement is not ideal (boxes will likely continue to parses old TLS).